Test/NUnitSetupClass.cs

```diff
@@ -10,11 +11,11 @@ public class NUnitSetupClass
             get {
                 var testDirectory = TestContext.CurrentContext.TestDirectory;
 
-                // trim off everything after '\\bin'
-                var index = testDirectory.IndexOf("\\bin");
+                // trim off everything after 'Path.DirectorySeparatorCharPath.DirectorySeparatorCharbin'
```

Comment:
This comment looks real odd, perhaps a char between Charbin? For example Char + bin?